packages/features/settings/layouts/SettingsLayoutAppDirClient.tsx

```diff
@@ -0,0 +1,788 @@
+"use client";
```

Comment:
This is a copy of `SettingsLayout` except that (1) this file has "use client" directive (as app router requires it for a client component) (2) the default component accepts data props like `currentOrg` and `otherTeams` (which will be passed from the server component `SettingsLayoutAppDir`